CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Upcoming
 
+### ⚠️ Important
+
+- Dependencies are no longer exposed (this includes Nuke and SwiftyGif). If you were using those dependencies we were exposing, you would need to import them manually.
```

Comment:
Maybe also add that they should "Reset package caches" after integrating? I had issues until I did that.

Reply:
Done 👍